packages/react-is/src/ReactIs.js

```diff
@@ -70,6 +72,7 @@ export const Portal = REACT_PORTAL_TYPE;
 export const Profiler = REACT_PROFILER_TYPE;
 export const StrictMode = REACT_STRICT_MODE_TYPE;
 export const Suspense = REACT_SUSPENSE_TYPE;
+export const SuspenseList = REACT_SUSPENSE_LIST_TYPE;
```

Comment:
Can we avoid exposing this and instead import the symbol directly in DevTools? It's not a stable API yet.